examples/ota-provider-app/ota-provider-common/OTAProviderExample.h

```diff
@@ -50,13 +51,33 @@ class OTAProviderExample : public chip::app::Clusters::OTAProviderDelegate
         kRespondWithBusy,
         kRespondWithNotAvailable
     };
+    static constexpr uint16_t SW_VER_STR_MAX_LEN = 32;
+    static constexpr uint16_t OTA_URL_MAX_LEN    = 512;
+    typedef struct DeviceSoftwareVersionModel
+    {
+        uint16_t vendorId;
+        uint16_t productId;
+        uint32_t softwareVersion;
+        char softwareVersionString[SW_VER_STR_MAX_LEN];
```

Comment:
```suggestion
        chip::CharSpan softwareVersionString;
```

Reply:
Isn't a span simply a view-only container? The actual array still has to exist somewhere in memory. Wouldn't it make more sense to simply have it in this struct as a char-array?

Reply:
@carol-apple given the 1/28 deadline, I would propose marking this comment as resolved and taking it up in a different ticket (if needed).